google-cloud-spanner/src/main/java/com/google/cloud/spanner/connection/ConnectionImpl.java

```diff
@@ -1234,6 +1372,19 @@ public ApiFuture<long[]> executeBatchUpdateAsync(Iterable<Statement> updates) {
     return internalExecuteBatchUpdateAsync(CallType.ASYNC, parsedStatements);
   }
 
+  private QueryOption[] mergeDataBoost(QueryOption... options) {
+    if (this.dataBoostEnabled) {
```

Comment:
Query - Just checking if its safe to not generate a query option when `dataBoostEnabled` is false? Wouldn't we want to differentiate the case when customer explicitly marked the option as `false` vs case where customer did not pass the property in connection string?